device/celestica/x86_64-cel_belgite-r0/CELESTICA-BELGITE/port_config.ini

```diff
@@ -0,0 +1,57 @@
+# name          lanes                 alias              index      speed     autoneg
+Ethernet0       26                    Ethernet1/0/1        1         1000      1
```

Comment:
We usually use `etp1` to `etpN`

Reply:
Sorry for the late reply, Blueve. Got it, I will check it will SDK guy and update it later.

Reply:
Blueve,
We did the change as you said . 
1. Change the alias to etp1 to etpN
2. Change the AN from 1/0 to on/off
Here is the log. Please review. I'll update later. Thanks.
[show-interface.log](https://github.com/Azure/sonic-buildimage/files/8095926/show-interface.log)